Source/Android/TealiumUnity/unity/src/main/java/com/tealium/EventEmitters.kt

```diff
@@ -5,6 +5,7 @@ import com.tealium.core.consent.ConsentManagementPolicy
 import com.tealium.core.consent.ConsentStatus
 import com.tealium.core.consent.UserConsentPreferences
 import com.tealium.core.messaging.UserConsentPreferencesUpdatedListener
+import com.tealium.remotecommands.BuildConfig
```

Comment:
Can remove, as you want to use the `BuildConfig.TAG` for this project, not the one for remote commands.